packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -97,7 +97,12 @@ abstract class TextSelectionControls {
   /// Builds a toolbar near a text selection.
   ///
   /// Typically displays buttons for copying and pasting text.
```

Comment:
I know the existing doc is already insufficient but can you add some doc for what each of the arguments should be?